src/main/kotlin/endpoint/RefundServerEndpoint.kt

```diff
@@ -58,10 +55,18 @@ class RefundServerEndpoint(
             })
     }
 
+    /**
+     * Creates a [EthRefundRequest] object from request string
+     */
+    private fun createRequest(txHash: String?): EthRefundRequest? {
+        // TODO replace destination ethereum address
+        return if (txHash != null) EthRefundRequest(txHash) else null
```

Comment:
Use Elvis operator

Reply:
yes, but how to call a method with elvis operator?